atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/HeartbeatServiceIntegrationTest.java

```diff
@@ -74,28 +66,24 @@ public void setUp() throws TException {
                                                 heartbeatTimePeriodMillis,
                                                 lockTable.getOnlyTable(),
                                                 writeConsistency);
-        clientPool.runWithRetry(this::createLockEntry);
+        String lockValue = CassandraTestTools.getHexEncodedBytes(
+                SchemaMutationLock.lockValueFromIdAndHeartbeat(lockId, 0));
+        CassandraTestTools.setLocksTableValue(clientPool, lockTable, lockValue, writeConsistency);
```

Comment:
Is there a reason why this doesn't just take in the lockId and heartbeat number?